conn_test.go

```diff
@@ -195,12 +210,133 @@ func TestConnectionPool_Broadcast(t *testing.T) {
 		},
 	}
 
-	cp.Broadcast(msg)
+	cp.BroadcastMessage(msg)
 
 	// wait until receive the msg
 	wg.Wait()
 }
 
+func TestConnectionPool_BroadcastMessages(t *testing.T) {
+	cp := cleisthenes.NewConnectionPool()
+
+	cAddress1 := cleisthenes.Address{"127.0.0.1", 8080}
```

Comment:
Port `8080` is often used. For example, tomcat(web server) uses `8080` as its default port.
So, I recommend to change the port number :)